blocks/library/paragraph/index.js

```diff
@@ -389,7 +389,7 @@ export const settings = {
 				},
 			},
 			save( { attributes } ) {
-				return attributes.content;
+				return <RawHTML key="html">{ attributes.content }</RawHTML>;
```

Comment:
Is key necessary here?

Reply:
>Is key necessary here?

Nope. Mistaken copy from the `migrate` function.